lib/src/model/game/game_repository.dart

```diff
@@ -104,4 +104,19 @@ class GameRepository {
       mapper: LightArchivedGame.fromServerJson,
     );
   }
+
+  Future<void> bookmark(GameId id, {int v = -1}) async {
+    // if v is -1, toggle the bookmark value on server
+    // otherwise explicitly set the new value
```

Comment:
If think `-1` is a bad idea, better to always explicitly set the new value, than to assume the current server state.

Reply:
Yes, this a temporary solution. When the `bookmarked` field is added I will remove the `-1`.